### PR TITLE
Add a naive regex fallback

### PR DESCRIPTION
**Pull request by omrilotan**
This changeset adds a fallback for engines that do not support lookbehind in regular expressions. This fallback is less accurate but catches ~78% of bots with less than 1% false-positive.
It is also exposed as "isbotNaive" export.

Solves #236

### CODE REVIEW

File: src/patterns.json [hunk 1]
```diff
@@ -11,7 +11,6 @@
   "(?<![hg]m)score",
   "@[a-z]",
   "\\(at\\)[a-z]",
-  "\\(github\\.com/",
```
Comment (jlowcs) [inline src/patterns.json:14]: These changes seem out of scope for this fix?

Comment (omrilotan) [reply src/patterns.json:14]: True, this is a side effect of my creating the naive pattern. I examined parts of the existing regular expression and replaced them with more efficient patterns that allow the same coverage.

**Review by jlowcs — APPROVED** [review]
